libs/akita-ng-entity-service/src/lib/ng-entity-service.config.ts

```diff
@@ -12,6 +13,7 @@ export interface NgEntityServiceGlobalConfig {
     PUT: HttpMethod;
     DELETE: HttpMethod;
   }>;
+  connector?: () => Subject<{ method: HttpMethod; loading: boolean; storeName: string; entityId?: any }>;
```

Comment:
Why you are not using the Event type?

Reply:
Fixed. -- Event wasn't being exported and shares the same name as Event from the DOM library. So I wasn't sure if there was a reason I wasn't aware of that it should only be used within the NgEntityServiceLoader. I actually put Event at first but then second-guessed myself 😅.